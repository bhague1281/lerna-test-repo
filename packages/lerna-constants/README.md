# `@bhague1281/lerna-constants`

> TODO: description

## Usage

```
const lernaConstants = require('@bhague1281/lerna-constants');

// TODO: DEMONSTRATE API
```
