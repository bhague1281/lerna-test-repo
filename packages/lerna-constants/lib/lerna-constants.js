"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.TEST3 = exports.TEST2 = exports.TEST1 = void 0;
var TEST1 = 'test1';
exports.TEST1 = TEST1;
var TEST2 = 'test2';
exports.TEST2 = TEST2;
var TEST3 = 'test3';
exports.TEST3 = TEST3;