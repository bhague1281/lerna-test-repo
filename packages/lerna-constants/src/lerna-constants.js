export const TEST1 = 'test1';
export const TEST2 = 'test2';
export const TEST3 = 'test3';
export const TEST4 = 'test4';
