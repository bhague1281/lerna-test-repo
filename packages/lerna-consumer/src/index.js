import {lernaConsumer} from './lerna-consumer';

export {
  lernaConsumer
};
