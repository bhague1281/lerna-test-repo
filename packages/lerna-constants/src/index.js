import * as LERNA_CONSTANTS from './lerna-constants';

export {
  LERNA_CONSTANTS
};
