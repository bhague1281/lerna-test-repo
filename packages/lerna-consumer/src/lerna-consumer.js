import {TEST1, TEST2, TEST3} from '@bhague1281/lerna-constants';

export const lernaConsumer = () => {
  return {
    meta: {
      info: 'This is a lerna consumer consuming constants (test)'
    },
    TEST1,
    TEST2,
    TEST3
  };
};
