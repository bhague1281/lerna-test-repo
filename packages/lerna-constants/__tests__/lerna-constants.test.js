import * as LERNA_CONSTANTS from '../src/lerna-constants';

describe('@bhague1281/lerna-constants', () => {
  it('should export the correct constants', () => {
    expect(LERNA_CONSTANTS).toMatchSnapshot();
  });
});
