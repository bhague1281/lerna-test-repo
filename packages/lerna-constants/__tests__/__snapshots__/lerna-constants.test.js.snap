// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@bhague1281/lerna-constants should export the correct constants 1`] = `
Object {
  "TEST1": "test1",
  "TEST2": "test2",
  "TEST3": "test3",
  "TEST4": "test4",
}
`;
